### PR TITLE
priojeet/feat: show get trading button when loggedin

### PR DESCRIPTION
**Pull request by priojeet-deriv**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by AbdallaMourad — APPROVED** [review]

File: src/components/layout/nav.js [hunk 1]
```diff
@@ -357,9 +366,16 @@ const NavMobile = ({ is_ppc, is_ppc_redirect }) => {
             </LogoLinkMobile>
             <MobileRight>
                 <LanguageSwitcher short_name="true" is_high_nav />
-                <MobileLogin margin_left="0.8rem" onClick={handleLogin} primary>
-                    <span>{localize('Log in')}</span>
-                </MobileLogin>
+
```
Comment (yashim-deriv) [inline src/components/layout/nav.js:369]:
```suggestion
```

File: src/components/layout/nav.js [hunk 2]
```diff
@@ -558,9 +574,17 @@ const NavDesktop = ({ base, is_ppc, is_ppc_redirect }) => {
                     has_scrolled={has_scrolled}
                 >
                     <LanguageSwitcher short_name="true" is_high_nav />
-                    <LoginButton onClick={handleLogin} primary>
-                        <span>{localize('Log in')}</span>
-                    </LoginButton>
+
```
Comment (yashim-deriv) [inline src/components/layout/nav.js:577]:
```suggestion
```

File: src/components/layout/nav.js [hunk 3]
```diff
@@ -558,9 +574,17 @@ const NavDesktop = ({ base, is_ppc, is_ppc_redirect }) => {
...
+                    {isLoggedIn() ? (
+                        <NowrapButton onClick={handleGetTrading} primary>
+                            <span>{localize('Get Trading')}</span>
+                        </NowrapButton>
+                    ) : (
+                        <NowrapButton onClick={handleLogin} primary>
+                            <span>{localize('Log in')}</span>
+                        </NowrapButton>
+                    )}
+
```
Comment (yashim-deriv) [inline src/components/layout/nav.js:587]:
```suggestion
```

**Review by yashim-deriv — APPROVED** [review]
A little nitpick, maybe can remove these whitespaces. Other than that, LGTM

**Review by suthesan-deriv — APPROVED** [review]

File: src/components/layout/nav.js (hunk 1 above)
Comment (priojeet-deriv) [reply src/components/layout/nav.js]: done :pray:

**Review by yashim-deriv — APPROVED** [review]

File: src/components/layout/nav.js [hunk 4]
```diff
@@ -607,15 +607,26 @@ const NavDesktop = ({ base, is_ppc, is_ppc_redirect }) => {
 }
 
 export const Nav = ({ base, is_ppc_redirect, is_ppc }) => {
+    const [is_logged_in, setLoggedIn] = useState(false)
+
+    useEffect(() => {
+        setLoggedIn(isLoggedIn())
+    }, [])
```
Comment (sean-binary) [inline src/components/layout/nav.js:614]: Would you want the `useEffect` to watch on the `is_logged_in` state and render accordingly? Or should the behaviour be only on the refresh of the page?

Comment (mustofa-binary) [reply src/components/layout/nav.js:614]: atm, i dont think there is a way to watch on cookies atm, let me know if you found smt otherwise

**Pull request merged**